packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -288,6 +289,20 @@ const ruleset: any = {
       },
     },
 
+    // RPC Code: RPC-Get-V1-11
+    AllowNestedIfParentExist: {
+      description:
```

Comment:
> description: [](http://example.com/codeflow?start=5&length=13)

set the flag for staging

Reply:
[https://github.com/Azure/azure-openapi-validator/blob/main/CONTRIBUTING.md#how-to-set-a-spectral-rule-to-run-only-in-staging](https://github.com/Azure/azure-openapi-validator/blob/main/CONTRIBUTING.md#how-to-set-a-spectral-rule-to-run-only-in-staging)

Reply:
Added